package.json

```diff
@@ -25,9 +28,10 @@
     "registry": "https://registry.npmjs.org"
   },
   "scripts": {
-    "build": "yarn clean && microbundle -f cjs,es",
-    "clean": "rm -rf dist",
+    "build": "yarn clean && microbundle -f cjs,es src/{index,server}.ts && mv dist/server.* . && yarn fix-server-imports",
+    "clean": "rm -rf dist server.*",
     "dev": "microbundle watch",
+    "fix-server-imports": "replace-in-files --string='\"./' --replacement='\"./dist/' server.* && replace-in-files --string='\"../' --replacement='\"./' server.*",
```

Comment:
I had to move server files to root to allow `@swan-io/chicane/server` import (until https://devblogs.microsoft.com/typescript/announcing-typescript-4-7-beta/#package-json-exports-imports-and-self-referencing is widely available)